cypress/integration/v2/blocks/container.spec.js

```diff
@@ -43,3 +48,256 @@ function switchLayout() {
+			[ `padding-top` ]: '120px',
+			[ `padding-bottom` ]: '120px',
+		},
+	} )
+	cy.adjust( 'Height', 'half' ).assertComputedStyle( {
+		'.ugb-container__wrapper': {
+			[ `min-height` ]: '50vh',
+		},
+	} )
+	cy.adjust( 'Height', 'full' ).assertComputedStyle( {
+		'.ugb-container__wrapper': {
+			[ `min-height` ]: '100vh',
+		},
+	} )
+
+	// Test Content Vertical Align
+	cy.adjust( 'Content Vertical Align', 'flex-start' ).assertComputedStyle( {
+		'.ugb-container__wrapper': {
+			[ `justify-content` ]: 'flex-start',
+		},
+	} )
+	cy.adjust( 'Content Vertical Align', 'center' ).assertComputedStyle( {
+		'.ugb-container__wrapper': {
+			[ `justify-content` ]: 'center',
+		},
+	} )
+	cy.adjust( 'Content Vertical Align', 'flex-end' ).assertComputedStyle( {
+		'.ugb-container__wrapper': {
+			[ `justify-content` ]: 'flex-end',
+		},
+	} )
+
+	// Test Content Width
+	cy.adjust( 'Content Width (%)', 68 ).assertComputedStyle( {
+		'.ugb-container__content-wrapper': {
+			[ `width` ]: '68%',
+		},
+	} )
+
+	// Test Content Horizontal Align
+	cy.adjust( 'Content Horizontal Align', 'flex-start' ).assertComputedStyle( {
+		'.ugb-container__side': {
+			[ `align-items` ]: 'flex-start',
+		},
+	} )
+	cy.adjust( 'Content Horizontal Align', 'center' ).assertComputedStyle( {
+		'.ugb-container__side': {
+			[ `align-items` ]: 'center',
+		},
+	} )
+	cy.adjust( 'Content Horizontal Align', 'flex-end' ).assertComputedStyle( {
+		'.ugb-container__side': {
+			[ `align-items` ]: 'flex-end',
+		},
+	} )
+
+	// Test Text Align
+	assertAligns( 'Align', '.ugb-inner-block', { viewport } )
+
+	// Container Tab
+	cy.collapse( 'Container' )
+
+	// Background Tab
+	desktopOnly( () => {
+		// Test Single
+		cy.adjust( 'Background', {
+			[ `Color Type` ]: 'single',
+			[ `Background Color` ]: '#000000',
+			[ `Background Color Opacity` ]: 0.7,
+		} ).assertComputedStyle( {
+			'.ugb-container__wrapper': {
+				[ `background-color` ]: 'rgba(0, 0, 0, 0.7)',
+			},
+		} )
+
+		// Test Gradient
+		cy.adjust( 'Background', {
+			[ `Color Type` ]: 'gradient',
+			[ `Background Color #1` ]: '#ff5c5c',
+			[ `Background Color #2` ]: '#7bff5a',
+			[ `Adv. Gradient Color Settings` ]: {
+				[ `Gradient Direction (degrees)` ]: 160,
+				[ `Color 1 Location` ]: 28,
+				[ `Color 2 Location` ]: 75,
+				[ `Background Gradient Blend Mode` ]: 'hue',
+			},
+		} ).assertComputedStyle( {
+			'.ugb-container__wrapper:before': {
+				[ `background-image` ]: 'linear-gradient(160deg, #ff5c5c 28%, #7bff5a 75%)',
+				[ `mix-blend-mode` ]: 'hue',
+			},
+			'.ugb-container__wrapper': {
+				[ `background-color` ]: '#ff5c5c',
+			},
+		} )
+	} )
+
+	// Test Image
+	cy.setBlockAttribute( {
+		[ `column${ viewport === 'Desktop' ? '' : viewport }BackgroundMediaUrl` ]: Cypress.env( 'DUMMY_IMAGE_URL' ),
+	} )
+
+	desktopOnly( () => {
+		cy.adjust( 'Background', {
+			[ `Background Media Tint Strength` ]: 7,
+			[ `Fixed Background` ]: true,
+			[ `Adv. Background Image Settings` ]: {
```

Comment:
Adv. Background Image Settings options are responsive except the Image Blend Mode option. let's pass the `viewport` option and move it outside the `desktopOnly` function